### PR TITLE
exposing the protos in a new repo

### PR DESCRIPTION
**Pull request by AsafEini**

### CODE REVIEW

File: com/coralogix/v1/metrics-view.proto [hunk 1]
```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+import "common.proto";
+import "google/protobuf/wrappers.proto";
+
+package metricViewPackage;
```
[the remark this answers is not in the document]
Comment (zhrebicek) [reply com/coralogix/v1/metrics-view.proto:5]: We also use `snake_case` for naming proto files and suffix `_service` for files containing service.

You should also get inspired here and possibly add audit logs if it is something exposed to customers in the furue, if not ignore me with it 
Example in service: https://github.com/coralogix/cx-api-tags/blob/master/com/coralogix/tags/v1/tag_external_service.proto#L18
Example of extension definition:  https://github.com/coralogix/cx-api-tags/blob/master/com/coralogix/tags/v1/audit_log.proto

File: buf.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: v1
+deps:
+  - buf.build/googleapis/googleapis
+lint:
+  use:
+    - DEFAULT
+  except:
```
Comment (zhrebicek) [inline buf.yaml:7]: I did not quite understood why it is not possible in TypeScript to just add wrappers/unwrappers of the Request & Response, could you please repeat it here when I am fresh today :) 

Also hooking @iravid here to throw some arguments against loosing up the linter if he has some.

File: com/coralogix/monitoring/v1/metrics_view_service.proto [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  google.protobuf.StringValue user_id = 5;
+  repeated Group groups = 6;
+  google.protobuf.StringValue base_metric_name = 7;
+}
+
+message AggregateResponse {
+  repeated TreeNode result = 1;
+}
+
+message TreeNode {
```
Comment (zhrebicek) [inline com/coralogix/monitoring/v1/metrics_view_service.proto:25]: I would throw these files to tree_node.proto to keep the service have only service and Req/Resp

File: com/coralogix/monitoring/v1/common.proto [hunk 4]
```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```
Comment (zhrebicek) [inline com/coralogix/monitoring/v1/common.proto:1]: Instead of `common` I would name it `group` as the "root message"

File: buf.yaml (hunk 2 above)
Comment (AsafEini) [reply buf.yaml:7]: after brainstorming with you i now believe we should stick to the opinionated buf linter rules.

**Review by zhrebicek — APPROVED** [review]

File: com/coralogix/monitoring/v1/monitoring_views_service.proto [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+message UpdateViewResponse {
+    MonitoringViewResponse result = 1;
+}
+
+message CreateViewResponse {
+  MonitoringViewResponse result = 1;
+}
+
+message CreateViewRequest {
+  MonitoringView payload = 1;
```
[the remark this answers is not in the document]
Comment (AsafEini) [reply com/coralogix/monitoring/v1/monitoring_views_service.proto:46]: what would you preferred?

[the remark this answers is not in the document]
Comment (zhrebicek) [reply com/coralogix/monitoring/v1/monitoring_views_service.proto:46]: `monitoring_view`

[the remark this answers is not in the document]
Comment (AsafEini) [reply com/coralogix/monitoring/v1/monitoring_views_service.proto:46]: sure thing!

**Pull request merged**